tests/onpage/test-class-onpage.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+* @package WPSEO\Unittests
+*/
+
+class WPSEO_OnPage_Double extends WPSEO_OnPage {
+
+	/**
+	 * Overwrite the request_indexibility method, because it uses a dependency
+	 *
+	 * @return int
+	 */
+	protected function request_indexability() {
+		if ( home_url() === 'http://example.org' ) {
```

Comment:
I will set home_url to `http://example.org`, to be sure the right value is used.